.gitmodules

```diff
@@ -58,7 +58,7 @@
 	url = https://github.com/oneapi-src/oneDNN.git
 [submodule "tools/pot/thirdparty/open_model_zoo"]
 	path = thirdparty/open_model_zoo
-	url = https://github.com/openvinotoolkit/open_model_zoo.git
+	url = https://github.com/mmikolajcz/open_model_zoo.git
```

Comment:
We need to remember to change it back to the origin PyBind repository after @jiwaszki's changes are released.